app/utils/__tests__/gameUtils.test.ts

```diff
@@ -54,16 +54,19 @@ describe('determineWinner', () => {
 });
 
 describe('hasFlor', () => {
-  it('returns true when all cards have the same suit', () => {
+  const muestra = { number: 4, palo: 'espada' as Palo };
```

Comment:
It's not realistic to have the muestra card to be also one of the drawn cards to a player. Please use something different here (while keeping the suit). _7 of espada_ would work.